### PR TITLE
Set default rep-type

### PR DESCRIPTION
**Pull request by holdenhinkle**
Resolves [ticket #68614](https://app.zenhub.com/workspaces/accredited-representation-management-team-64d0dc51d3e8f4788ac6ef96/issues/gh/department-of-veterans-affairs/va.gov-team/68614)

## Summary
Set's a default for the "Search for a representative" rep-type. Some clean up.

## Testing done
None.

We're developing the 'Find a Representative' app based on approved wireframes but no user-testing has been conducted yet. User-testing is happening in parallel with the development of the app. Once UX Research comes back with the user-testing results, we'll update the app accordingly and write tests.

In short, we don't want to write tests for an implementation that may very well change based on user-testing.

## What areas of the site does it impact?
None. This app is behind a feature flag.

### CODE REVIEW

File: src/applications/representative/components/search/RepTypeSelector.jsx [hunk 1]
```diff
@@ -12,26 +9,27 @@ const RepTypeSelector = ({ onChange }) => {
   return (
     <>
       <div className="rep-type-selector">
-        <VaRadio
+        <va-radio
```
Comment (jvcAdHoc) [inline src/applications/representative/components/search/RepTypeSelector.jsx:12]: What's the difference between these two? I mostly see `VaRadio` used and a mix of `VaRadioOption` vs `va-radio-option`.

**Review by jvcAdHoc — APPROVED** [review]

**Review by cosu419 — APPROVED** [review]
Built locally and no regression issues. Approved!

File: src/applications/representative/components/search/RepTypeSelector.jsx (hunk 1 above)
Comment (rmessina1010) [reply src/applications/representative/components/search/RepTypeSelector.jsx:12]: React bound components are useful when you need to pass event listeners or other  such functions via props.

**Review by rmessina1010 — APPROVED** [review]
Looks good to me

**Comment by jknipes** [comment]
Closing ticket out, [see Slack thread for details](https://dsva.slack.com/archives/CBU0KDSB1/p1698870225955309?thread_ts=1698768818.600699&cid=CBU0KDSB1)
